From 9dbf440cda69cfa3496b6c4e4ad4c46121035b14 Mon Sep 17 00:00:00 2001
From: Dimitrios Apostolou <dimitrios.apostolou@cfengine.com>
Date: Fri, 13 Jan 2017 18:59:32 +0100
Subject: [PATCH 06/15] _XOPEN_SOURCE 500, i.e. XPG5, can't be used with C99
 compiler.

It was useless anyway, srandom() and random() were not used anywhere.
---
 libraries/liblmdb/lmdump.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libraries/liblmdb/lmdump.c b/libraries/liblmdb/lmdump.c
index 0e3ee4b..9065468 100644
--- a/libraries/liblmdb/lmdump.c
+++ b/libraries/liblmdb/lmdump.c
@@ -6,7 +6,6 @@
      -d : print only the size of keys and values
  */
 
-#define _XOPEN_SOURCE 500        /* srandom(), random() */
 #include <stdio.h>
 #include <stdlib.h>
 #include <time.h>
-- 
2.38.1

